bootstrap/kubeadm/main.go

```diff
@@ -34,6 +34,7 @@ import (
 	kubeadmbootstrapv1alpha2 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha2"
 	kubeadmbootstrapv1alpha3 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha3"
 	kubeadmbootstrapcontrollers "sigs.k8s.io/cluster-api/bootstrap/kubeadm/controllers"
+	"sigs.k8s.io/cluster-api/cmd/version"
```

Comment:
It "feels" a bit weird to import `cmd/version` here, I'm wondering if the implementation should be moved somewhere else in the tree?

Reply:
@detiber what do you suggest? move to `util`?

Reply:
Maybe open an issue to tackle this during the next iteration, we shouldn't move exported packages in a v0.3.x release

I'm fine using this package for now

Reply:
created issue here https://github.com/kubernetes-sigs/cluster-api/issues/2823